app/models/cloud_object_store_object/operations.rb

```diff
@@ -1,7 +1,7 @@
 module CloudObjectStoreObject::Operations
   extend ActiveSupport::Concern
 
-  def delete_cloud_object_store_object
+  def cloud_object_store_object_delete
```

Comment:
looks good. do we need to keep (deprecated) old link in there for a while?

Reply:
@romanblanco - bump

Reply:
```
manageiq-ui-classic/spec/controllers/application_controller/ci_processing_spec.rb
215:          "delete_cloud_object_store_object",
225:          :task   => "delete_cloud_object_store_object",
228:        controller.send(:process_objects, [object1.id, object2.id], "delete_cloud_object_store_object", "delete")
235:          :task   => "delete_cloud_object_store_object",
292:          "delete_cloud_object_store_object",
302:          :task   => "delete_cloud_object_store_object",
305:        controller.send(:process_objects, [object.id.to_s], "delete_cloud_object_store_object", "delete")
312:          :task   => "delete_cloud_object_store_object",
```

Those :)